test/integration/app/prompted_nobuild.js

```diff
@@ -312,20 +313,68 @@ describe('Prompt and no build integration tests for app generator', function ()
         "/persons": {
           "get": {
             "description": "Gets `Person` objects.",
+            "responses": {
+              "200": {
+                "description": "pet response",
+                "schema": {
+                  "$ref": "#/definitions/Person"
+                }
+              }
+            }
```

Comment:
After ending up with a mess in some of the prior tests, we've been trying to add only the minimum necessary to the test fixtures for what the test needs to run. I'm surprised that the generator doesn't fail on invalid Swagger input since I thought we ran it through a Swagger validator. The intent is to keep the test very focused. I'm fine with a bit more in the Swagger, provided it is actually needed for the test, so another option is to check we are validating properly and/or that we are producing the right output to correspond to this extra input data.